### DISCUSSION TITLE
客户端打开后文件显示无法连接，网页端打开文件也会显示报错

### DISCUSSION THREAD

**Discussion by liunjiajun** [Moonvy/figmaEX#40] [Q&A]
客户端：
一打开后显示弹窗：
![微信截图_20240715092427](https://github.com/user-attachments/assets/4205495e-0fa3-46f7-b867-be686178cfe5)
打开文件后：
![微信图片_20240715092545](https://github.com/user-attachments/assets/f8e0468c-d805-42e5-a5a5-8e3337740a4b)

网页端：
进入界面后打开文件无法打开：
![微信截图_20240715092639](https://github.com/user-attachments/assets/76b7214c-fed1-4a6c-9dfe-c73b23924178)

**Comment by yArna** [comment]
1.下载安装最新版本试试

2. 是否打开了「禁用自动网络代理」功能，切换试试
<img width="473" alt="image" src="https://github.com/user-attachments/assets/30bb5923-3444-40f6-8b71-65a8422b4d8e">
